src/components/views/rooms/BasicMessageComposer.js

```diff
@@ -152,6 +152,7 @@ export default class BasicMessageEditor extends React.Component {
         if (this.props.placeholder) {
             const {isEmpty} = this.props.model;
             if (isEmpty) {
+                this._formatBarRef.hide();
```

Comment:
We probably also want to do this if there is no placeholder. Perhaps move isEmpty and this check outside of the parent if?

Reply:
@bwindels oops! I didn't think of that. Making changes to a commit now. Thanks.